### ISSUE TITLE
Localizing extensions guide

### ISSUE THREAD

**Issue by KamasamaK** [microsoft/vscode-docs#5774]
Now that extension localization is built into the core of VS Code, it would be nice to have a proper localization guide for extensions.

**Comment by gregvanl** [comment]
@TylerLeonhardt

**Comment by TylerLeonhardt** [comment]
Agreed! You can find a good overview over in https://github.com/Microsoft/vscode-l10n but a guide on the website would be great
